test/functional/ui/screen.lua

```diff
@@ -205,6 +205,8 @@ end
 
 function Screen:try_resize(columns, rows)
   uimeths.try_resize(columns, rows)
+  -- Give ourselves a chance to _handle_resize
+  self:sleep(0.1)
```

Comment:
That's essentially what the sleep call is doing, but it also has the `notification_cb` set so we get notified about the `resize`, in order to call `_handle_resize`.